test/update.test.js

```diff
@@ -0,0 +1,203 @@
+'use strict';
+
+var testHelpers = require('./test_helpers');
+
+describe('record updates', function () {
+  var airtable;
+  var teardownAsync;
+
+  beforeAll(function () {
+    return testHelpers.getMockEnvironmentAsync().then(function (env) {
+      airtable = env.airtable;
+      teardownAsync = env.teardownAsync;
+    });
+  });
+
+  afterAll(function () {
+    return teardownAsync();
+  });
+
+  describe('non-destructive updates', function () {
+    it('can update one record', function () {
+      return airtable
+        .base('app123')
+        .table('Table')
+        .update('rec123', {
+          foo: 'boo',
+          bar: 'yar',
+        })
+        .then(function (updatedRecord) {
```

Comment:
Should we also add a few tests with callbacks instead of promises?

Reply:
Done here and in #94.